programs/survival/fakeGui.sc

```diff
@@ -0,0 +1,417 @@
+// 原作者: Scarpet
+// 修改: 猴子
+// 建議: 企鵝
+
+__config() -> {
+    'scope' -> 'global',
+    'stay_loaded' -> true,
+    'requires' -> {
+        'carpet' -> '>=1.4.57'
+    },
+    'commands' -> {
+        '' -> _() -> print('\n使用方式:\n  對這假人按下右鍵\n或是:\n  執行指令/fakeGui <player>'),
+        '<player>' -> _(fakeplayer) -> (
```

Comment:
I think that since most people will be using this in English, it's best to put these messages in English, and maybe add an additional language setting.

Reply:
3Q~~

Reply:
啊，你得知道3在中文里的发音了，才能知道3Q是thank you的谐音......
对方不懂中文，你这样说他不一定看得懂的......

Reply:
痾.... 好尷尬ww感謝提醒ww